README.md

```diff
@@ -12,20 +12,20 @@ $ go install ./...
 
 ## Tools
 
-### traced
+### collectd
```

Comment:
I propose renaming `traced` to `collectd`. The original name is misleading because this component does not perform tracing.

Reply:
I disagree. `collected` is so generic as to be meaningless.
What does it collect? Why traces, so it's a trace daemon.